site/docs/spec.md

```diff
@@ -254,6 +255,33 @@ Notes:
 
 1. Technically, data files can be deleted when the last snapshot that contains the file as “live” data is garbage collected. But this is harder to detect and requires finding the diff of multiple snapshots. It is easier to track what files are deleted in a snapshot and delete them when that snapshot expires.
 
+#### Deletion Files
```

Comment:
This section should be "Position-based delete files" to distinguish these from equality delete files that will be added later.

Reply:
Make sense!